core/schema_utils.py

```diff
@@ -288,10 +298,16 @@ def normalize_against_schema(
             for validator in schema[SCHEMA_KEY_VALIDATORS]:
                 kwargs = dict(validator)
                 del kwargs['id']
-                assert get_validator(
-                    validator['id'])(normalized_obj, **kwargs), (
-                        'Validation failed: %s (%s) for object %s' % (
-                            validator['id'], kwargs, normalized_obj))
+                validator_func = get_validator(validator['id'])
+                if validator_func.__name__ == 'has_subtitled_html_non_empty':
```

Comment:
Errr... let's not hardcode this. Do it similar to `id` something around:
```                
                kwargs = dict(validator)
                expect_invalid_default_value = kwargs['expect_invalid_default_value']
                del kwargs['id']
                del kwargs['expect_invalid_default_value']
                if (
                    not get_validator(validator['id'])(normalized_obj, **kwargs)
                    and expect_invalid_default_value
                ):
                    raise AssertionError(.....)
```

Reply:
Updated